src/appleseed.shaders/include/appleseed/color/as_color_helpers.h

```diff
@@ -188,7 +188,7 @@ float as_luminance(color in_C, string colorspace, string illuminant)
     {
         Y = as_luminance_D60(in_C, colorspace);
     }
-    else if (illuminant = "D65")
```

Comment:
Woops, the OSL compiler doesn't warn in this case?

Reply:
@dictoon no, at least i didn't noticed it.